recipes/pdspy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+package:
```

Comment:
Please use jinja templating for the name and version as in:

https://github.com/conda-forge/staged-recipes/tree/master/recipes/example